shipyard-controller/handler/sequencedispatcher_test.go

```diff
@@ -357,6 +358,146 @@ func TestSequenceDispatcher_AddError(t *testing.T) {
 
 }
 
+func TestSequenceDispatcher_QueueIsNotEmpty(t *testing.T) {
```

Comment:
Can we also add a test case where there are currently no sequences with state = `started`, but other queued sequences that preceed the current one?

Reply:
such a test is there, pushed it few minutes earlier, we can have a call to explain if you want :)